### PR TITLE
UHM-3337 -- Patient Lab Page

### PR DESCRIPTION
**Pull request by rhenshaw56**

### CODE REVIEW

File: src/rest/orderRest.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { axiosInstance } from '../config';
+
+const api = {
+  fetchOrdersByPatient: patientUUID => axiosInstance.get(`order?totalCount=true&sort=desc&status=active&patient=${patientUUID}&v=full`)
```
Comment (mogoodrich) [inline src/rest/orderRest.js]: I just noticed that this restricts to status=active, so let's change the name to fetchActiveOrdersByPatient

File: src/rest/encounterRest.js [hunk 2]
```diff
@@ -44,6 +44,16 @@ const api = {
       }
     }),
 
+  // used in lab-workflow
+  getPatientEncounters: ({ patientUUID, encounterTypeUUID }) => axiosInstance.get(`encounter?s=default&patient=${patientUUID}&encounterType=${encounterTypeUUID}&v=full`)
```
Comment (mogoodrich) [inline src/rest/encounterRest.js]: @rhenshaw56 I'm not sure exactly how this differs from the fetchEncountersByPatient besides the "s=default" and the representation type (the v parameter). (I'm not sure if s=default actually does anything?)  What were the problems you were encountering when you tried using the fetchEncountersByPatient method?  

If you need  a different REST representation than what is provided by DEFAULT_ENCOUNTER_REP I think you can either add additional parameters to DEFAULT_ENCOUNTER_REP or have fetchPatientByEncounters accept a third (optional) parameter called representation that allows one you override the DEFAULT_ENCOUNTER_REP value.

Comment (rhenshaw56) [reply src/rest/encounterRest.js]: @mogoodrich when I tried using the `fetchEncountersByPatient` rest call, I was getting this on postman.
screen shot 2018-10-24 at 16 49 54

Comment (mogoodrich) [reply src/rest/encounterRest.js]: thanks @rhenshaw56 , can you include the rest of the error?

Comment (mogoodrich) [reply src/rest/encounterRest.js]: It's possible I broke something changing the DEFAULT_REST_REP...

**Pull request merged**